### PR TITLE
Cleanup and bump to 1.0.4

### PR DESCRIPTION
**Pull request by kwilcox**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -49,12 +33,11 @@ def readme():
       long_description=readme(),
       license='Apache License 2.0',
       author="Luke Campbell",
-      author_email="lcampbell@asascience.com",
+      author_email="luke.campbell@gdit.com",
```
Comment (lukecampbell) [inline setup.py]: @Bobfrat you should probably change this to your email. I am just a contributor now, not the maintainer.

Comment (Bobfrat) [inline setup.py]: @kwilcox can you please change to robert.fratantonio@rpsgroup.com?

**Comment by Bobfrat** [comment]
So changing the author to me broke the build huh? Guess it wasn't meant to be

**Comment by kwilcox** [comment]
conda/s3/something is having issues atm. I'm getting failed builds all over the place.

**Comment by kwilcox** [comment]
@Bobfrat can you release on pypi? I'll bump conda.

**Comment by Bobfrat** [comment]
Ok should be good https://pypi.python.org/pypi/cc-plugin-glider

**Pull request merged**